chore: sync managed files from governance template

The following managed files have drifted from the canonical source in `f5xc-salesdemos/docs-control`:

- CLAUDE.md
- .github/workflows/dependabot-auto-merge.yml
- biome.json
- zizmor.yaml
- .claude/governance.json

This issue was created automatically by the governance enforcement workflow.

## CI Failure Report — PR #202 blocked on verify_governance tests

**PR:** #202 (`governance/sync-managed-files`)
**Head SHA:** `fcff26ebff8bfdec6a7e8ee75ff712cef6a2ce04` (updated from main to pick up PR #204's verify_governance sync-branch exemption)
**Failed check:** `Run pytest suite` ([run 24821680557](https://github.com/f5xc-salesdemos/api-specs-enriched/actions/runs/24821680557/job/72647657619)) — 4m13s
**Total CI duration:** ~5.5 min (Contract-diff gate skipped because pytest failed)

### Root cause

PR #204 added a sync-branch exemption to `scripts/verify_governance.py`:

```
ok: skipping governance check on sync branch governance/sync-managed-files
```

This short-circuits the verify script with exit code 0 whenever the head branch is `governance/sync-managed-files`. The two failing tests set up fixtures that simulate exactly that branch context and expect the pre-exemption behavior:

- `tests/test_verify_governance.py::TestMain::test_exit_one_when_violation` — expects `rc == 1` on violation, actually got `rc == 0`
- `tests/test_verify_governance.py::TestMain::test_exit_two_on_missing_manifest` — expects `rc == 2` on missing manifest, actually got `rc == 0`

Both tests were **passing before PR #204** because the sync-branch exemption did not exist. PR #204 updated the production code but did not update the affected tests.

### Failing assertion excerpts

```
tests/test_verify_governance.py:184: AssertionError
E       assert 0 == 2
Captured stdout: ok: skipping governance check on sync branch governance/sync-managed-files
```

```
FAILED tests/test_verify_governance.py::TestMain::test_exit_one_when_violation - assert 0 == 1
FAILED tests/test_verify_governance.py::TestMain::test_exit_two_on_missing_manifest - assert 0 == 2
```

Suite result: `2 failed, 1967 passed, 2 xfailed in 157.62s`

### Remediation options

1. Update the two tests to force a non-sync branch in the fixture (e.g., set `--head` to `feature-x` instead of `feature`, or set an env var the fixture reads to bypass the exemption).
2. Add a testability hook in `scripts/verify_governance.py` (e.g., `--no-sync-exemption` flag or `VERIFY_GOVERNANCE_DISABLE_SYNC_EXEMPTION=1` env) and use it in the two tests.
3. Refactor the exemption into a separately-tested helper so the `main()` tests can exercise the violation/missing-manifest paths independently.

### Impact on release chain

This PR (#202) is the first gate in the release-unblocking chain:
- #202 (this PR) — sync biome 2.4.10 → 2.4.12 — **BLOCKED on pytest**
- #205 (orphan release) — waiting on biome 2.4.12 on main before Super-Linter passes
- Separate timeout-fix PR for `sync-and-enrich.yml` `WAIT_FOR_MERGE_MAX_TOTAL=1800` — not yet opened, queued behind #202

Nothing downstream can proceed until the pytest tests are corrected.

### Action required

Fix the two failing tests and push to the `governance/sync-managed-files` branch. CI will auto-re-run.

--- a/.claude/governance.json
+++ b/.claude/governance.json
@@ -1,7 +1,18 @@
 {
   "source_repo": "f5xc-salesdemos/docs-control",
   "skip_files": {
-    "xcsh": ["biome.json", "LICENSE", "CONTRIBUTING.md", "ruff.toml", ".ruff.toml", ".python-lint", ".mypy.ini"]
+    "xcsh": [
+      "biome.json",
+      "LICENSE",
+      "CONTRIBUTING.md",
+      "ruff.toml",
+      ".ruff.toml",
+      ".python-lint",
+      ".mypy.ini",
+      ".github/workflows/super-linter.yml"
+    ],
+    "api-specs": ["ruff.toml", ".ruff.toml", ".python-lint", ".mypy.ini"],
+    "api-specs-enriched": ["ruff.toml", ".ruff.toml", ".python-lint", ".mypy.ini"]
   },
   "protected_files": [
     ".github/workflows/github-pages-deploy.yml",

--- a/.github/workflows/dependabot-auto-merge.yml
+++ b/.github/workflows/dependabot-auto-merge.yml
@@ -1,5 +1,6 @@
 ---
 name: Dependabot Auto-Merge
+
 on: pull_request_target
 
 permissions:
@@ -7,22 +8,7 @@ permissions:
   pull-requests: write
 
 jobs:
-  auto-merge:
-    runs-on: ubuntu-latest
+  call:
     if: github.actor == 'dependabot[bot]'
-    steps:
-      - name: Fetch Dependabot metadata
-        id: metadata
-        uses: dependabot/fetch-metadata@v2
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-      - name: Approve PR
-        run: gh pr review --approve "$PR_URL"
-        env:
-          PR_URL: ${{ github.event.pull_request.html_url }}
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: Enable auto-merge
-        run: gh pr merge --auto --squash "$PR_URL"
-        env:
-          PR_URL: ${{ github.event.pull_request.html_url }}
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    uses: f5xc-salesdemos/docs-control/.github/workflows/dependabot-auto-merge.yml@main
+    secrets: inherit

--- a/CLAUDE.md
+++ b/CLAUDE.md
@@ -13,8 +13,13 @@ Never run `git commit`, `git push`, `gh pr create` directly.
 ```
 Agent(
   subagent_type="f5xc-github-ops:github-ops",
+  mode="bypassPermissions",
   prompt="<type>: <desc>\n\nFiles:\n- <list>\n\nWhy: <reason>"
 )
 ```
+
+`mode="bypassPermissions"` is required — without it, plan mode can
+re-engage mid-workflow and strip the agent's Bash access, leaving
+it stuck after the first step.
 
 See `CONTRIBUTING.md` for project rules.

--- a/biome.json
+++ b/biome.json
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://biomejs.dev/schemas/2.4.10/schema.json",
+  "$schema": "https://biomejs.dev/schemas/2.4.12/schema.json",
   "formatter": {
     "indentStyle": "space",
     "indentWidth": 2,

--- a/zizmor.yaml
+++ b/zizmor.yaml
@@ -1,11 +1,8 @@
 ---
 rules:
-  # pull_request_target required for
-  # dependabot-auto-merge and require-linked-issue
-  pull-request-target:
-    disable: true
-  # We use version tags (@v6, @v8) for
-  # first-party GitHub actions, not SHAs
+  # SHA-pinning is enforced by convention and by ref-version-mismatch.
+  # This rule's heuristic flags some valid SHA pins; redundant with
+  # the stricter rule we already rely on.
   unpinned-uses:
     disable: true
   # Low-confidence finding;

